src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,23 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# Feel free to change the signature. Min ID is the next available Mondo ID.
+slurp/%.tsv: components/%.owl tmp/mondo.sssom.tsv reports/mirror-signature-mondo.tsv | slurp/
```

Comment:
@Nico: One thing I don't like about `make` is that it's sometimes hard to get answers to questions on Google. What does the pipe `|` do in the list of dependencies?

Reply:
Google for “make dependencies pipe” - it's a so called order only dependency, which means that its only triggered if at least one other dependency (normal dependency) was updated.

Reply:
Thanks. That actually came up w/ [something](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html). I'll give a read. I googled something like "make what is the pipe | at end of dependencies", but didn't have any luck.

Reply:
I understand how this works now and why it's there. I read: http://gnu.org/software/make/manual/html_node/Prerequisite-Types.html

I think it's saying something different than what you're saying, though. It says: 
> Occasionally, however, you have a situation where you want to impose a specific ordering on the rules to be invoked without forcing the target to be updated if one of those rules is executed. In that case, you want to define order-only prerequisites.

Also I feel like "order-only prerequisite" is a very strange name for this. Don't understand what 'order' has to do with it; I would have called the ones on the left of `|` something like "target update prerequisites" and the right "non target update prerequisites" or "target update independent prerequisites".